src/py42/sdk/queries/fileevents/filters/cloud_filter.py

```diff
@@ -3,23 +3,49 @@
 
 
 class Actor(FileEventFilterStringField):
+    """Class that filters events by the cloud service username of the event originator
+    (applies to cloud data source events only).
+    """
+
     _term = u"actor"
 
 
 class DirectoryID(FileEventFilterStringField):
+    """Class that filters events by unique identifier of the cloud drive or folder where the event
+     occurred (applies to cloud data source events only).
+    """
+
     _term = u"directoryId"
 
 
 class Shared(QueryFilterBooleanField):
+    """Class that filters events by the shared status of the file at the time the event occurred
+    (applies to cloud data source events only).
+    """
+
     _term = u"shared"
 
 
 class SharedWith(FileEventFilterStringField):
+    """Class that filters events by the users files were explicitly shared with at the time of the
```

Comment:
Missing word _that_  and apostrophe?

"Class that filters events by the user's files _that_ were explicitly..."